SUSYBSMAnalysis/HSCP/test/UsefulScripts/DeDxStudy/DeDxStudy.C

```diff
@@ -179,7 +182,11 @@ void DeDxStudy(string DIRNAME="COMPILE", string INPUT="dEdx.root", string OUTPUT
    gStyle->SetNdivisions(505,"X");
    TH1::AddDirectory(kTRUE);
 
-   bool isData = INPUT.find("MC")!=string::npos ? false : true;
+   bool isData   = !(INPUT.find("MC")      !=string::npos) ,
```

Comment:
I told you several times to proceed differently and to call the signal sample per datasetName, so it uses Analysis_Samplte.txt information